tests/unit/utils/test_log.py

```diff
@@ -254,6 +254,28 @@ def test_python_warnings_werror(self, args):
         with pytest.raises(PendingDeprecationWarning):
             warnings.warn("test warning", PendingDeprecationWarning)
 
+    @pytest.mark.parametrize(
+        'console_cli, console_conf, console_expected, ram_conf, ram_expected',
+        [
+            (None, None, logging.INFO, None, logging.NOTSET),
+            (None, None, logging.INFO, 'CRITICAL', logging.CRITICAL),
+            (None, 'WARNING', logging.WARNING, 'INFO', logging.INFO),
+            ('INFO', 'WARNING', logging.INFO, 'VDEBUG', logging.VDEBUG),
+            ('WARNING', 'INFO', logging.WARNING, 'CRITICAL', logging.CRITICAL),
+        ])
+    def test_init_from_config(self, mocker, console_cli, console_conf,
```

Comment:
```suggestion
    def test_init_from_config(self, console_cli, console_conf,
```

as it's not used anymore.